detox/test/e2e/t-pasteboard.js

```diff
@@ -0,0 +1,41 @@
+describe('Pasteboard', () => {
+  beforeEach(async () => {
+    await device.reloadReactNative();
+  });
+
+  beforeEach(async () => {
+    await element(by.text('Pasteboard')).tap();
+  });
+
+  it('should have stringTextInput', async () => {
+    await element(by.id('stringValueInput')).typeText('exampleString\n');
+    await element(by.id('CheckButton')).tap();
+    await device.pasteboardInfo();
```

Comment:
I don't understand. What does this check? This returns a JSON with the pasteboard info...—and?

Reply:
So basically I see missing API to `expect` on the pasteboard. `pasteboardInfo()` should be an internal function, not external.